app/controllers/brexit_checker_controller.rb

```diff
@@ -34,6 +34,9 @@ def results
     all_actions = BrexitChecker::Action.load_all
     @criteria = BrexitChecker::Criterion.load_by(criteria_keys)
     @actions = filter_items(all_actions, criteria_keys)
+    @audience_actions = @actions.group_by(&:audience)
```

Comment:
this variable does not seem to be used anywhere so does not have to be an instance variable.

Reply:
changed back to a normal variable